src/core/events/display.rs

```diff
@@ -0,0 +1,52 @@
+use std::fmt;
+
+#[allow(dead_code)] // FIXME
+#[derive(Default, Clone, Copy)]
+pub(crate) enum DisplayFormat {
+    SingleLine,
+    #[default]
+    MultiLine,
```

Comment:
BTW, I found a bit confusing that the default value on this enum is `Multiline` but on the CLI it's `SingleLine`. Why is that?